src/ImageRunModal.jsx

```diff
@@ -103,10 +103,17 @@ export class ImageRunModal extends React.Component {
 
     constructor(props) {
         super(props);
-        let command = "sh";
+
+        let command = "";
         if (this.props.image && this.props.image.Command) {
             command = utils.quote_cmdline(this.props.image.Command);
         }
+
+        let entrypoint = "";
+        if (this.props.image && this.props.image.Entrypoint) {
+            entrypoint = utils.quote_cmdline(this.props.image.Entrypoint);
+        }
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1155-20230104-111228-5fe35f89-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Same thing -- please add a test, and use `?.` if you want to.